spec/system/sake_index_order_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe "Sake Index Order" do
+  describe "sakes order in index" do
+    let!(:sealed1) { create(:sake, name: "未開封のお酒1", bottle_level: "sealed") }
+    let!(:opened1) { create(:sake, name: "開封済みのお酒1", bottle_level: "opened") }
+    let!(:empty) { create(:sake, name: "空のお酒", bottle_level: "empty") }
+    let!(:sealed2) { create(:sake, name: "未開封のお酒2", bottle_level: "sealed") }
+    let!(:opened2) { create(:sake, name: "開封済みのお酒2", bottle_level: "opened") }
+
+    before do
+      visit sakes_path
+    end
+
+    context "if not included empty bottle" do
```

Comment:
```suggestion
    context "if the 'show empty bottles' checkbox is unchecked" do
```
if not included empty bottle だと、空の酒が登録されていない場合だと誤解しそう。

Reply:
not checkedで採用しました